src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  height: 16px;
```

Comment:
```suggestion
```
No need for this. You have given the heigh for your `star` element

Reply:
From the very beginning I tried not to use "height: 16" but my test showed an error http://joxi.ru/ZrJK3jDhbNJQlm. So then I took this solution from another students repository and somehow it got fixed